utils/ALA_dialogues.cfg

```diff
@@ -365,11 +365,11 @@
     {MOVE_UNIT (id=Divaid,Croquet) 8 13}
     [message]
         speaker=Glauco
-        message= _ "What's the situation, "+$trapper_name+"?"
+        message= _ "What's the situation?"
     [/message]
-    {SCOUT_MESSAGE ( _ "I would say it's bad, but I'm afraid it's awful; The old caravan station was partially swallowed up by the river, and it seems that a group of Nagas have nestled there.")}
+    {SCOUT_MESSAGE ( _ "I would say it's bad, but I'm afraid it's awful. The old caravan station was partially swallowed up by the river, and it seems that a group of Nagas have nestled there.")}
```

Comment:
This one does seem like it ought to be a period, given that the "The" is already capitalized, so this change is okay.